src/Controller/Api/GalleryController.php

```diff
@@ -245,7 +245,9 @@ public function getGalleryMediasAction($id): array
      *
      * @param int|string $id Gallery identifier
      *
-     * @return Collection<array-key, GalleryItemInterface>
+     * @return Collection<int|string, GalleryItemInterface>
+     *
+     * @phpstan-return Collection<array-key, GalleryItemInterface>
```

Comment:
What do we win with this over the `@return` ?

Reply:
Fixed